app/routes.py

```diff
@@ -1,2 +1,326 @@
-from flask import Blueprint
+# datetime is a Python module - use to work with dates as date objects
+import datetime
+# the Python requests library is the de facto standard for formatting HTTP requests in Python, which you use in conjuction with a request method a la requests.get() or requests.post()
+import requests
+#  Response is a Flask class that represents HTTP responses - note that make_response() actually uses Response internally
+from flask.wrappers import Response 
+# need to import db module in order to add and commit (save) to databases when requests get made to add or update database items
+from app import db
+# Both models need to be imported to use them to link code to their respective databases (a Model is just a class inherited from the db.Model)
+from app.models.task import Task
+from app.models.goal import Goal
+# asc and desc are module functions that need to be imported to be used
+from sqlalchemy import asc, desc
+# The following are all dependencies:
+# request is an OBJECT (not to be confused with the 'requests' PACKAGE) used to get info about an HTTP request
+# Blueprint is a class 
+# make_response is a Flask helper method that instantiates a Response object 
+# jsonify is a method used to convert a JSON HTTP request body into a Python dictionary 
+from flask import request, Blueprint, make_response, jsonify
+# dotenv is a Python package 
+from dotenv import load_dotenv
+# os is a module that provides the ability to read environmental variables, which are actually stored outside of the code - needed to be able to read the hidden bot API token
+import os
+
+from flask import abort
+
+# this method loads the values from the .env file which the os module can then read 
+load_dotenv()
+
+# instantiating blueprints for task and goal
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+# HELPER FUNCTIONS:
+#=============================================================================
+
+def slack_post_to_task_notifications_channel(text):
```

Comment:
Love that you made this its own helper function! That helps make the code easier to read. :)